tests/integration/helpers/hlr-lookup.spec.js

```diff
@@ -46,7 +46,7 @@ describe('Integration: HLR Lookup with callback', function () {
     return getTimeoutPromise(500)
     .then(function () {
```

Comment:
Maybe arrow function, but maybe this will be done in a refactoring branch. I'm not sure

Reply:
when refactoring